plugins/chezmoi/chezmoi.plugin.sh

```diff
@@ -0,0 +1,8 @@
+# any future aliases can be added here
```

Comment:
```suggestion
#! bash oh-my-bash.module

# any future aliases can be added here
```

What are *future* aliases? Aren't *added aliases* trivially all future ones?

Reply:
chezmoi got plenty of commands, for now I have added the ones that I have used the most.

I could add all the commands too, but not having used them - it's a bit tricky to come up with aliases for them.

Was thinking to add them gradually or other folks could contribute it over time.

Reply:
Anyway, the first line should be updated.

For the code comment `any future aliases can be added here`, you can just remove it unless you are working on adding the aliases of the other commands of `chezmoi`. Or, if you're working on it, you many replace it with something like `additional aliases will be added here`.